vnext/ReactWindowsCore/ReactWindowsCore-Desktop.vcxproj

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(SolutionDir)\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props')" />
```

Comment:
are we sure we want WinRT in Desktop?
@aeulitz Will this cause issues?

Reply:
We need C++/WinRT in Desktop. For instance, the WebSocket module is implemented with such APIs.

Note, C++/WinRT can be projected into "classic" Win32 apps, as long as the underlying API is available in that Windows version.

Reply:
We currently support Windows 8.1 in desktop. Please make sure things work fine on that OS version.